humans/models.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.auth.models import AbstractUser
+from django_auth_ldap.backend import LDAPBackend, _LDAPUser
+from humans.auth.hashers import make_ldap_password
+import ldap
+
+
+class User(AbstractUser):
+    def save(self, *args, **kwargs):
+        is_new = self.id is None
+        if is_new:
+            self.__ldap__save()
+        password = self._password
+        super().save(*args, **kwargs)
+        if password is not None and not is_new:
+            self.__ldap__set_password(password)
+
+    def __ldap__save(self):
+        # ldap_user = _LDAPUser(LDAPBackend(), username=)
```

Comment:
@txels This is the code supposed to allow creating users, but I left it commented for some reason. Probably it didn't work or I didn't finish it.